doc/classes/ColorPickerButton.xml

```diff
@@ -6,6 +6,7 @@
 	<description>
 		Encapsulates a [ColorPicker] making it accessible by pressing a button. Pressing the button will toggle the [ColorPicker] visibility.
 		See also [BaseButton] which contains common properties and methods associated with this node.
+		Note: By default the button may not be wide enough for the color preview swatch to be visible. To workaround this issue you can type space characters into the [Text] property or change the value of the [rect_min_size] property.  
```

Comment:
```suggestion
		[b]Note:[/b] By default the button may not be wide enough for the color preview swatch to be visible. Make sure to set [member Control.rect_min_size] to a big enough value to give the button enough space. 
```

Reply:
I'd also add a comma:

`By default, the button ...`